main.py

```diff
@@ -88,10 +103,32 @@ async def read_archive(date: str = (datetime.date.today() - datetime.timedelta(d
 async def read_latest():
     token = get_token_from_main_page()
     currencies, _, _ = get_prices_from_api(token)
-    prices = {c.code.lower(): {"sell": c.sell, "buy": c.buy} for c in currencies}
+    prices = {c.code.lower(): {"sell": c.sell, "buy": c.buy}
+              for c in currencies}
     return prices
 
 
+@app.get("/archive/range")
+@cache(expire=60 * 60 * 24)
+async def customrange(startDate: str = (datetime.date.today() - datetime.timedelta(days=1)).strftime("%Y-%m-%d"), endDate: str = datetime.date.today().strftime("%Y-%m-%d")):
```

Comment:
I suggest this name for method.
Also I think it's not necessary to set default value for start date.
```suggestion
async def read_archive_range(start_date: str, end_date: str = datetime.date.today().strftime("%Y-%m-%d")):
```